docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -9,104 +9,48 @@ You need a Kubernetes or Openshift cluster, and the `kubectl` or `oc` command-li
 
 == Step 1. Deploy Hazelcast Platform Operator
 
-To deploy Hazelcast Platform Operator, apply the https://repository.hazelcast.com/operator/bundle-{full-version}.yaml[`bundle-{full-version}.yaml`] file to your Kubernetes cluster:
+Starting from the Hazelcast Platform Operator 5.6.0, a Helm chart is available to install the operator. It is available from the Hazelcast Helm Charts repository and can be added to your Helm repository list by running the following command:
 
-[tabs]
-====
-Kubernetes::
-+
---
-[source,shell,subs="attributes+"]
-----
-kubectl apply -f https://repository.hazelcast.com/operator/bundle-{full-version}.yaml
-----
---
-Openshift::
-+
---
-[source,shell,subs="attributes+"]
+[source,shell]
 ----
-oc apply -f https://repository.hazelcast.com/operator/bundle-{full-version}.yaml
+helm repo add hazelcast https://hazelcast-charts.s3.amazonaws.com/
+helm repo update
 ----
---
-====
-
-TIP: If you always want to use the latest release, you can use the `bundle-latest.yaml` file.
 
-At this point, the Hazelcast Platform Operator should be up and running. You can check it with the command below.
+=== Deploy the Hazelcast Platform Operator and the CRDs together
```

Comment:
We don't really use headings under steps. I'd suggest adding a numbered step and then two substeps, something like this...

. Deploy the Hazelcast Platform Operator. You can either deploy the Hazelcast Platform Operator at the same time as CRDs are separately. Deploying CRDs separately avoids installing any cluster-scoped resources.
.. Run the following command to.....
.. Run the following command to